src/library/Uno.Material/Styles/Controls/v2/TextBox.xaml

```diff
@@ -8,10 +8,48 @@
 					xmlns:ut="using:Uno.Themes"
 					mc:Ignorable="d macos">
 
+	<!--
+		Expected TextBox behavior:
+		We are going a different route than the Official Material M2/M3 Google samples & the Material doc regarding the fact that the Placeholder will always be visible.
+		(cf. specs discussions here https://github.com/unoplatform/Uno.Themes/issues/974)
+		
+		- Without Header/Placeholder, we will have a simple TextBox (matching Windows behavior)
+		
+		- With a Placeholder and no Header, we will have a simple TextBox (matching Windows behavior).
```

Comment:
"simple TextBox (matching Windows behavior)" please be more precise in the comment
it may be obvious when you are working on it, but not everyone is

placeholder will be display where the text is supposed to be, until any text is entered